Resources/public/css/views/fields/edit/country.css

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
as far as I can see, `views/fields/edit/country.css` and `theme/views/fields/edit/country.css` are almost the same as the selection.css files. To avoid that and since CountryEditView extends SelectionEditView, you should probably force the container of the CountryEditView instance to get the class `ez-view-selectioneditview` in addition to `ez-view-countryeditview`.